RSFormView.podspec

```diff
@@ -2,7 +2,7 @@
 Pod::Spec.new do |s|
 
   s.name         = "RSFormView"
-  s.version      = "2.1.3"
+  s.version      = "2.1.4"
```

Comment:
there's no need to bump version on each PR, we can do that once hackathon work is all done since I'm guessing you aren't pushing a new tag and cocoapod version to go with it.

Reply:
right